### PR TITLE
disable ipv6 solicitations in mininet

### PR DESCRIPTION
**Pull request by eth0xFEED**
while debugging an issue noticed that kernel originated packets filling the logs , this turns off that behavior

### CODE REVIEW

File: tools/mininet/stratum.py [hunk 1]
```diff
@@ -276,6 +276,8 @@ def config(self, **params):
             cmd = "/sbin/ethtool --offload %s %s off" \
                   % (self.defaultIntf(), off)
             self.cmd(cmd)
+        cmd = "sysctl net.ipv6.conf.%s.disable_ipv6=1" % (self.defaultIntf())
+        self.cmd(cmd)
```
Comment (pudelkoM) [inline tools/mininet/stratum.py:280]: I'm not sure how the `hosts` export for mininet works, but shouldn't this command be it's own class? Like `NoIpv6Host`?

Comment (eth0xFEED) [reply tools/mininet/stratum.py:280]: Good point will get that fixed up 👍

File: tools/mininet/stratum.py [hunk 2]
```diff
@@ -278,8 +278,20 @@ def config(self, **params):
...
+
 
 # Exports for bin/mn
 switches = {'stratum-bmv2': StratumBmv2Switch}
 
-hosts = {'no-offload-host': NoOffloadHost}
+hosts = {
+    'no-offload-host': NoOffloadHost,
+    'no-ipv6-host': NoIpv6OffloadHost
+    }
```
Comment (pudelkoM) [inline tools/mininet/stratum.py:297]:
```suggestion
}
```

**Review by pudelkoM — DISMISSED** [review]
LGTM, but I'll let @Yi-Tseng decide. He has more experience with mininet.

**Review by Yi-Tseng — DISMISSED** [review]
LGTM

**Review by Yi-Tseng — APPROVED** [review]

File: tools/mininet/stratum.py [hunk 3]
```diff
@@ -278,8 +278,20 @@ def config(self, **params):
...
+
 
 # Exports for bin/mn
 switches = {'stratum-bmv2': StratumBmv2Switch}
 
-hosts = {'no-offload-host': NoOffloadHost}
+hosts = {
+    'no-offload-host': NoOffloadHost,
+    'no-ipv6-host': NoIpv6OffloadHost
+}
```
Comment (pudelkoM) [inline tools/mininet/stratum.py:297]: @Yi-Tseng do you know how this export interacts with Mininet? Does this create two hosts?

Comment (Yi-Tseng) [reply tools/mininet/stratum.py:297]: This is for the command line parameter, for example:

`mn --custom stratum.py --switch stratum-bmv2 --host no-ipv6-host linear,3`

Ref:
https://github.com/stratum/stratum/blob/main/tools/mininet/Dockerfile#L118

**Pull request merged**